File: 공통문제/8주차/음양더하기/편근형/PKH.java
class Solution {
    public int solution(int[] absolutes, boolean[] signs) {
        int answer = 0;
        int len = absolutes.length;
        for (int i = 0; i < len; i++) {
            if (signs[i] == true) {
                answer += absolutes[i];
            }
            else {
                answer -= absolutes[i];
            }
        }
        return answer;
    }
}

File: 공통문제/8주차/음양더하기/PKH.py
def solution(absolutes, signs):
    answer = 0
    for i,j in zip(absolutes, signs):
        if j == False:
            answer -= i
        else:
            answer += i
    return answer


# sign에 따라서 '+', '-'를 정해 반환값에 더하거나 빼는 방식 
